tests/routing_table/test_routing_table_ordered_covering.py

```diff
@@ -0,0 +1,364 @@
+import pytest
+
+from rig.routing_table import (
+    RoutingTableEntry, Routes, table_is_subset_of, MinimisationFailedError
+)
+from rig.routing_table.ordered_covering import (
+    get_generality, get_all_merges, get_insertion_index, Merge, refine_merge,
+    minimise
+)
+
+
+@pytest.mark.parametrize(
+    "key, mask, generality", (
+        (0x0, 0x0, 32),
+        (0x1, 0x1, 31),
```

Comment:
Would be worth either testing or documenting either way. (I'd prefer the output to be valid/sensible if such an entry exists anyway...).